debezium-core/src/main/java/io/debezium/connector/common/BaseSourceTask.java

```diff
@@ -138,6 +138,15 @@ public final void start(Map<String, String> props) {
                     LOGGER.info("   {} = {}", propName, propValue);
                 });
             }
+            try {
+                this.coordinator = start(config);
+                setTaskState(State.RUNNING);
+            }
+            catch (RetriableException e) {
+                LOGGER.warn("Failed to start connector, will re-attempt during polling.", e);
+                restartDelay = ElapsedTimeStrategy.constant(Clock.system(), retriableRestartWait.toMillis());
```

Comment:
I think there is a method that takes Duration as parameter.

Reply:
right, fixed